src/components/proposalBooking/bookingSteps/BookingEventStep.tsx

```diff
@@ -152,7 +152,7 @@ export default function BookingEventStep({
     handleRowsChange((rows) => [
       ...rows,
       {
-        id: `t-${Date.now()}`,
+        id: Date.now(),
```

Comment:
Maybe the id could be `proposalBooking.id` instead of time

Reply:
The thing here is to add just a temporary id because when you add new lost time or new time slot on the proposal booking it doesn't have id at that time. After you save the id is returned by the backend and everything is fine.